app/assets/stylesheets/govuk_publishing_components/components/_checkboxes.scss

```diff
@@ -15,14 +27,29 @@
 
 .gem-c-checkboxes__list {
   padding-left: govuk-spacing(0);
+  margin: 0;
+
+  .gem-c-checkboxes__list {
```

Comment:
I'm not sure this block is needed.

In the only place where I can see a variant with nested lists, "Checkbox items with nested checkboxes", the child list doesn't have this class. That variant seems to use the `govuk-checkboxes--nested` class for its padding and margin.